diff --git a/src/Sylius/Bundle/AdminBundle/EmailManager/OrderEmailManager.php b/src/Sylius/Bundle/AdminBundle/EmailManager/OrderEmailManager.php
index 4eaff957874..60d2decf5e8 100644
--- a/src/Sylius/Bundle/AdminBundle/EmailManager/OrderEmailManager.php
+++ b/src/Sylius/Bundle/AdminBundle/EmailManager/OrderEmailManager.php
@@ -16,6 +16,7 @@
 use Sylius\Bundle\CoreBundle\Mailer\Emails;
 use Sylius\Component\Core\Model\OrderInterface;
 use Sylius\Component\Mailer\Sender\SenderInterface;
+use Webmozart\Assert\Assert;

 final class OrderEmailManager implements OrderEmailManagerInterface
 {
@@ -25,14 +26,17 @@ public function __construct(private SenderInterface $emailSender)

     public function sendConfirmationEmail(OrderInterface $order): void
     {
+        $email = $order->getCustomer()->getEmail();
+        Assert::notNull($email);
+
         $this->emailSender->send(
             Emails::ORDER_CONFIRMATION_RESENT,
-            [$order->getCustomer()->getEmail()],
+            [$email],
             [
                 'order' => $order,
                 'channel' => $order->getChannel(),
                 'localeCode' => $order->getLocaleCode(),
-            ]
+            ],
         );
     }
 }
diff --git a/src/Sylius/Bundle/AdminBundle/EmailManager/ShipmentEmailManager.php b/src/Sylius/Bundle/AdminBundle/EmailManager/ShipmentEmailManager.php
index 5270cc23123..02f6b26fcf7 100644
--- a/src/Sylius/Bundle/AdminBundle/EmailManager/ShipmentEmailManager.php
+++ b/src/Sylius/Bundle/AdminBundle/EmailManager/ShipmentEmailManager.php
@@ -17,6 +17,7 @@
 use Sylius\Component\Core\Model\OrderInterface;
 use Sylius\Component\Core\Model\ShipmentInterface;
 use Sylius\Component\Mailer\Sender\SenderInterface;
+use Webmozart\Assert\Assert;

 final class ShipmentEmailManager implements ShipmentEmailManagerInterface
 {
@@ -28,12 +29,18 @@ public function sendConfirmationEmail(ShipmentInterface $shipment): void
     {
         /** @var OrderInterface $order */
         $order = $shipment->getOrder();
+        $email = $order->getCustomer()->getEmail();
+        Assert::notNull($email);

-        $this->emailSender->send(Emails::SHIPMENT_CONFIRMATION, [$order->getCustomer()->getEmail()], [
-            'shipment' => $shipment,
-            'order' => $order,
-            'channel' => $order->getChannel(),
-            'localeCode' => $order->getLocaleCode(),
-        ]);
+        $this->emailSender->send(
+            Emails::SHIPMENT_CONFIRMATION,
+            [$email],
+            [
+                'shipment' => $shipment,
+                'order' => $order,
+                'channel' => $order->getChannel(),
+                'localeCode' => $order->getLocaleCode(),
+            ],
+        );
     }
 }
diff --git a/src/Sylius/Bundle/AdminBundle/Event/OrderShowMenuBuilderEvent.php b/src/Sylius/Bundle/AdminBundle/Event/OrderShowMenuBuilderEvent.php
index 7e2cfde7545..da90d9fd52c 100644
--- a/src/Sylius/Bundle/AdminBundle/Event/OrderShowMenuBuilderEvent.php
+++ b/src/Sylius/Bundle/AdminBundle/Event/OrderShowMenuBuilderEvent.php
@@ -25,7 +25,7 @@ public function __construct(
         FactoryInterface $factory,
         ItemInterface $menu,
         private OrderInterface $order,
-        private StateMachineInterface $stateMachine
+        private StateMachineInterface $stateMachine,
     ) {
         parent::__construct($factory, $menu);
     }
